packages/datatrak-web/src/features/Questions/AutocompleteQuestion.tsx

```diff
@@ -22,8 +26,11 @@ const Autocomplete = styled(BaseAutocomplete)`
 
   .MuiFormLabel-root {
     color: ${({ theme }) => theme.palette.text.primary};
-    font-size: 1rem;
-    line-height: 1.5;
+    font-size: 0.875rem;
+    line-height: 1.2;
+    @media (min-width: ${MOBILE_BREAKPOINT}) {
```

Comment:
since we've matched up the MOBILE_BREAKPOINT const with the theme md break point, should we just use the theme everywhere?